packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -398,50 +399,69 @@ router.post(
   })
 );
 
-const cqLinkStatusSchema = z.enum(cqLinkStatus);
-
-const completeEnhancedCoverageSchema = z.object({
+const updateECAfterIncludeListSchema = z.object({
+  ecId: uuidSchema,
   cxId: uuidSchema,
-  patientIds: uuidSchema.array(),
-  cqLinkStatus: cqLinkStatusSchema,
+  patientIds: nonEmptyStringListFromQuerySchema,
+  cqOrgIds: nonEmptyStringListFromQuerySchema,
 });
 
 /** ---------------------------------------------------------------------------
- * POST /internal/patient/enhance-coverage/completed
+ * POST /internal/patient/enhance-coverage/after-include-list
  *
- * Indicate the coverage enhancement has been completed.
+ * Store the result of running Enhanced Coverage from the local environment.
  *
- * @param req.query.cxId The customer ID.
- * @param req.body.patientIds The IDs of patient to update their CQLinkStatus and trigger doc query.
- * @param req.body.cqLinkStatus The status of the link to CareQuality. (one of "linked",
- *                              "processing", "failed").
- * @return 200 OK - processing asynchronously
+ * @return 200 OK
  */
 router.post(
-  "/enhance-coverage/completed",
```

Comment:
Removing this b/c its not being used in the product now, it was an upcoming endpoint for EC through lambdas that ended up making its way in the original PR.